Taskfile.yml

```diff
@@ -47,8 +47,6 @@ tasks:
     sources:
       - package.json
       - yarn.lock
-    generates:
-      - node_modules/**
```

Comment:
comment: I don't know if this is as-intended by `task` or a bug, but if `generates:` is present here, it always re-runs the commands and doesn't cache as expected.
It seems like nothing currently relies on this `generates` specification, so I removed it to allow `task` to cache the frontend's `yarn install`.